test/README.md

```diff
@@ -155,6 +155,26 @@ import pdb; pdb.set_trace()
 anywhere in the test. You will then be able to inspect variables, as well as
 call methods that interact with the bitcoind nodes-under-test.
 
+If further introspection of the bitcoind instances themselves becomes
+necessary, this can be accomplished by first setting a pdb breakpoint
+at an appropriate location, running the test to that point, then using
+`gdb` to attach to the process and debug:
+
+```bash
```

Comment:
perhaps also mention that this may require superuser privileges, so may need `sudo gdb ...`